src/lib/api/container/client/containerApiClient.client.ts

```diff
@@ -110,3 +110,46 @@ export const deleteFood = async (
     return Err('API response is invalid');
   }
 };
+
+export interface IPostContainerRequestBody {
+  /**
+   * An identifier of a group that a new container will belong to
+   */
+  groupId: IGroup['id'];
+  /**
+   * New container name a user input
+   */
+  name: string;
+}
+
+export interface IPostContainerResponse {
+  /**
+   * An identifier of a newly created container
+   */
+  containerId: IContainer['id'];
+}
+
+/**
+ * Function to send a request to the API to create a new container
+ * @param requestBody A {@link IPostContainerRequestBody} object to be sent to API as the request body
+ * @returns A {@link IPostContainerResponse} object for success, an error message if fails
+ */
+export const postContainer = async (
```

Comment:
For clarity, could you rename it to `postCreateContainer`, please? Because there would likely to be multiple `POST` method functions.

I think I should have renamed the functions for foods, by the way...